### PR TITLE
[Docs Site] core formatting checks

### PR DESCRIPTION
**Pull request by Cherry**
### Summary

- Formats remaining files left over from previous formatting ones
- Adds a "core" formatting check for JS/TS, etc. that's part of the docs site

This also adds some scripts for formatting other content, but none of that runs by default yet and will likely need to wait for https://github.com/cloudflare/cloudflare-docs/issues/17011

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Great

**Review by KianNH — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -12,7 +12,11 @@
 		"check:functions": "npx tsc --noEmit -p ./functions/tsconfig.json",
 		"check:worker": "npx tsc --noEmit -p ./worker/tsconfig.json",
 		"dev": "npx astro dev",
-		"format": "npx prettier --write \"**/*.{js,jsx,ts,tsx,mjs,astro,css,json,yaml,yml}\"",
+		"format": "npm run format:core && npm run format:content && npm run format:data",
```
Comment (KianNH) [inline package.json]:
```suggestion
		"format": "npm run format:core && npm run format:data",
```

Comment (KianNH) [reply package.json]: We're still discussing if we want to use Prettier (or alternative formatters) for MD/MDX at the moment so I think we can remove this (for users of npm run format) but keep the format:content check around for our testing.

**Review by KianNH — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Cherry) [reply package.json]: Sure, sounds good!

**Pull request merged**